packages/react-component-playground/webpack.config.js

```diff
@@ -35,22 +35,22 @@ module.exports = {
     'react-dom': 'react-dom',
   },
   resolve: {
-    extensions: ['', '.js', '.jsx'],
+    extensions: ['.js', '.jsx'],
   },
   module: {
```

Comment:
`webpack` v2 wraps this in [rules](https://webpack.js.org/configuration/module/#module-rules) configuration.